native/cocos/editor-support/spine/Bone.cpp

```diff
@@ -198,10 +200,10 @@ void Bone::updateWorldTransform(float x, float y, float rotation, float scaleX,
             break;
         }
     }
-    _a *= _skeleton.getScaleX();
-    _b *= _skeleton.getScaleX();
-    _c *= _skeleton.getScaleY();
-    _d *= _skeleton.getScaleY();
```

Comment:
Refactor `Bone::updateWorldTransform`, save some directives by invoking the getter at the beginning of the function.